scripts/prepare_ci.sh

```diff
@@ -111,7 +111,11 @@ compile_circuit keccak256 keccak256_256_test.circom
 
 # Setup and compile rsa
 npm_install rsa
-compile_circuit rsa main.circom
+compile_circuit rsa main_rsa.circom
+
+# # Setup and compile anonAadhaar
+npm_install anonAadhaar
+compile_circuit anonAadhaar aadhaar-verifier.circom
```

Comment:
we skip anon aadhaar tests in CI because it takes too much time

Reply:
Right, I uncommented to build it and test witness generation.